### ISSUE TITLE
OpenGL Error Invalid Operation

### ISSUE THREAD

**Issue by PhilippII** [jbikker/lighthouse2#2]
I am just starting out with lighthouse2 trying to compile it on my laptop.
glGetError() in CheckGL() seems to be giving me INVALID_OPERATION

Could you please have a look at lines 118 - 120 in platform.cpp 
		glBindVertexArray( 0 );
		glDisableVertexAttribArray( 0 );
		glDisableVertexAttribArray( 1 );

I think there is something wrong. 

OpenGL reference says:
"GL_INVALID_OPERATION is generated by glEnableVertexAttribArray and glDisableVertexAttribArray if no vertex array object is bound."
But we are unbinding the vertex array object in glBindVertexArray(0) with the argument zero so there is no vao bound. Or did I get something wrong here?

**Comment by PhilippII** [comment]
Aahh sorry. Actually I think my laptop was using the integrated GPU instead of the actual NVIDIA GPU as stated in the troubleshooting page. I thought I had made sure to use the proper GPU. But apparently I did not. Switching to the NVIDIA GPU fixed my problem. Sorry my bad. Or maybe it was my recent gaphics driver update that helped too. I don't know. Anyway it's working now.

Should I close the issue now?
But still those 3 lines I mentioned are not quite correct according to the documentation of glDisableVertexAttribArray(0), are they? Although my driver does not give a warning anymore now. Should it not give a warning here according to the OpenGL reference? Correct me if I am wrong.
